provisions/roles/openshift/tasks/setup_mailservice.yml

```diff
@@ -21,3 +25,5 @@
       state: started
```

Comment:
should we also change the state of this container as of others, like `restarted` ?